src/shared-components/optionButton/style.ts

```diff
@@ -2,30 +2,47 @@ import styled from '@emotion/styled';
 import { css } from '@emotion/core';
 
 import { style as TYPOGRAPHY_STYLE } from '../typography';
-import { COLORS, ANIMATION, SPACER, BOX_SHADOWS } from '../../constants';
-import Container from '../container';
+import {
+  COLORS, ANIMATION, SPACER, BOX_SHADOWS, 
+} from '../../constants';
+import { containerStyles, ContainerType } from '../container/style';
 
-const getOptionTypeStyles = optionType => {
+import { OptionButtonProps } from '.';
+
+type BaseIconWrapperStylesProps = Required<
+  Pick<OptionButtonProps, 'buttonType' | 'optionType' | 'selected'>
+>;
+
+const getOptionTypeStyles = (
+  optionType: BaseIconWrapperStylesProps['optionType'],
+) => {
   if (optionType === 'checkbox') {
     return css`
       border-radius: 4px;
     `;
   }
+
   return css`
     border-radius: 50%;
   `;
 };
 
-const getTypeColor = type => {
-  if (type === 'secondary') {
+const getTypeColor = (buttonType: BaseIconWrapperStylesProps['buttonType']) => {
+  if (buttonType === 'secondary') {
     return COLORS.secondary;
   }
+
   return COLORS.primary;
 };
 
-export const ClickableContainer = styled(Container)`
+export const ClickableContainer = styled.button<{
+  containerType: ContainerType;
+}>`
+  ${({ containerType }) => containerStyles(containerType)};
   padding: ${SPACER.large};
   margin-bottom: ${SPACER.medium};
+  width: 100%;
+  text-align: left;
```

Comment:
`width` and `text-align` are new changes to account for button differences from existing div behavior.